static-assets/themes/cstudioTheme/css/console.css

```diff
@@ -116,6 +116,15 @@ input[type="checkbox"].content-type-selector--checkbox {
   margin-left: 239px;
 }
 
+#admin-console.embedded #categories-panel{
+  display: none;
+}
+
+#admin-console.embedded #cstudio-admin-console-workarea {
+  width: 100%;
+  margin-left: 0;
+}
```

Comment:
Doesn't look like you're using these yet.

Reply:
The class is being set to the `#admin-console` element